source/extensions/stat_sinks/wasm/config.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/server/instance.h"
+
+#include "server/configuration_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace StatSinks {
+namespace Wasm {
+
+/**
+ * Config registration for the Wasm statsd sink. @see StatSinkFactory.
+ */
+class WasmSinkFactory : Logger::Loggable<Logger::Id::config>,
```

Comment:
Please add unit tests for this type.

You can use `test/extensions/access_loggers/wasm/config_test.cc` as an example.